e2e/fixtures/components/dependency-status/all-files-in-component-map/b.js

```diff
@@ -0,0 +1 @@
+require('a.js');
```

Comment:
why do you need both folders? just make one of them, and in your tests, in one case add all and in another case don't add all